provision-cluster/lib/kubeception.js

```diff
@@ -72,9 +72,12 @@ async function createKluster(name, version) {
 
     if (response.message.statusCode == 200) {
       return await response.readBody()
+    } else if (response.message.statusCode == 425) {
```

Comment:
Could you add a comment about what this special case is?

Reply:
@AidanWenzel Yes, excellent point. I just pushed a commit with some more comments. Have a look and let me know if anything is unclear.